React/CoreModules/RCTWebSocketModule.mm

```diff
@@ -51,6 +51,16 @@ - (NSArray *)supportedEvents
   return @[ @"websocketMessage", @"websocketOpen", @"websocketFailed", @"websocketClosed" ];
 }
 
+
+- (void)flush
```

Comment:
this function will be called from here: https://github.com/discord/discord/pull/72585